src/components/DpDatepicker/DpDatepicker.vue

```diff
@@ -25,6 +25,11 @@ export default {
       default: 0
     },
 
+    dataCy: {
+      type: String,
+      required: false,
```

Comment:
if required false, we should add a default `datepicker`

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/443/commits/46a46e85b691c3ed777c8e06c4074b9ce53652e5